sound/effect.h

```diff
@@ -610,12 +610,12 @@ class Effect {
 #endif
 
 #ifdef ENABLE_SD
+    if (!LSFS::Begin()) ProffieOSErrors::sd_card_not_found();
```

Comment:
The No SD Card error doesn't show because it only happens at boot (right?).
Should this go here so it gets reported at preset change too?

Reply:
Good question, it might be useful, might be confusing if the SD card is busy (mounted computer) though.